cmd/aws-s3-csi-mounter/csimounter/csimounter.go

```diff
@@ -77,12 +80,24 @@ func Run(options Options) (int, error) {
 
 	exitCode, err := options.CmdRunner(cmd)
 	if err != nil {
-		// If Mountpoint fails, write it to `mountErrorPath` to let `PodMounter` running in the same node know.
-		if writeErr := os.WriteFile(mountErrorPath, stderrBuf.Bytes(), mountErrorFileperm); writeErr != nil {
-			klog.Infof("Failed to write mount error logs to %s: %v\n", mountErrorPath, err)
+		// If Mountpoint fails, write it to `options.MountErrPath` to let `PodMounter` running in the same node know.
+		if writeErr := os.WriteFile(options.MountErrPath, stderrBuf.Bytes(), mountErrorFileperm); writeErr != nil {
+			klog.Infof("Failed to write mount error logs to %s: %v\n", options.MountErrPath, err)
```

Comment:
Should probably be warn or error level

Reply:
Addressed with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/368/commits/eff45d72b5761636f1c3a42cfa03e0917cc6ec21